etc/local_grapl/grapl_provision.py

```diff
@@ -22,6 +28,11 @@
 from grapl_analyzerlib.schemas.risk_node_schema import RiskSchema
 from grapl_analyzerlib.schemas.schema_builder import ManyToMany
 
+def create_secret(secretsmanager):
+    secretsmanager.create_secret(
+        Name="JWT_SECRET_ID",
+        SecretString="jwt_secret",
```

Comment:
Let's make this a bit more representative. How about you generic a uuid v4, make it a string, and pass that in.